### ISSUE TITLE
Syntax error when using a "=" in an attribute

### ISSUE THREAD

**Issue by jwoertink** [jeromegn/slang#12]
`<script>` tags will sometimes have SRI checking with an `integrity` attribute like the latest minified version of [jQuery](https://code.jquery.com/). This attribute contains a sha256 hash which ends in a `=` causing slang to throw a syntax error

```slim
script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"
```

Or in short hand

```slim
h1 id="asdf=" Hello
```

```text
1. __kilt_io__ << "<h1"
2. __kilt_io__ << " id"
3. __kilt_io__ << "=\""
4. ("asdf=" Hello).to_s __kilt_io__
5. __kilt_io__ << "\""
6. __kilt_io__ << ">"
7. __kilt_io__ << "</h1>"
8. __kilt_io__ << "
9. "
--------------------------------------------------------------------------------
Syntax error in expanded macro: macro_4631584368:4: unexpected token: Hello
("asdf=" Hello).to_s __kilt_io__
```

Maybe this is related to #6 ?

**Comment by crisward** [comment]
I found this issue for this. It only happend when the `=` is at then end of the string. The lexer was saying, if = is before " then start parsing the value. However it wasn't checking if it was already parsing the value, so failed to close the value on the second quotes. 

I've fixed this added a test and included it in this pull request https://github.com/jeromegn/slang/pull/16

**Comment by jeromegn** [comment]
Should be fixed as of 1.4.2 with @crisward's PR.
